client/coral-admin/src/components/UserDetail.js

```diff
@@ -55,6 +58,28 @@ export default class UserDetail extends React.Component {
     }
   }
 
+  bulkAcceptThenReload = async () => {
+    try {
+      await this.props.bulkAccept();
+      this.props.data.refetch();
+    } catch (err) {
+
+      // TODO: handle error.
+      console.error(err);
+    }
+  }
+
+  bulkRejectThenReload = async () => {
+    try {
+      await this.props.bulkReject();
+      this.props.data.refetch();
+    } catch (err) {
+
+      // TODO: handle error.
+      console.error(err);
```

Comment:
Could you handle the error with a Notification? 

Look for `import {notify} from 'coral-framework/actions/notification';` in our codebase to find the implementation :)

Reply:
Sure, I will add the notification.  I based the bulkAcceptThenReload and  bulkRejectThenReload functions  in rejectThenReload and acceptThenReload functions. Those function does not have the notifiation neither, do you want that I add a notification there too?

Reply:
That would be terrific! `notify` is a redux action that needs to be mounted. Then just call `this.props.notify('error', getErrorMessages(err));`. `getErrorMessages` is in `import {getErrorMessages} from 'coral-framework/utils'`